src/mon/MonCommands.h

```diff
@@ -488,10 +488,9 @@ COMMAND("mon set_location " \
 COMMAND("mon enable_stretch_mode " \
 	"name=tiebreaker_mon,type=CephString, "
 	"name=new_crush_rule,type=CephString, "
-	"name=dividing_bucket,type=CephString, "
-	"name=pools,type=CephString,n=N",
+	"name=dividing_bucket,type=CephString, ",
 	"enable stretch mode, changing the peering rules and "
-	"failure handling on the named <pools> with <tiebreaker_mon> "
+	"failure handling on all pools with <tiebreaker_mon> "
```

Comment:
The data structures support setting this on individual pools and my initial draft had users specify the stretch pools when enabling stretch mode, but it opens up a whole category of new failure cases and UX issues by doing so that I thought we should leave out initially. For instance, when stretch mode is enabled on all pools and somebody creates a new pool, do we assume it's a stretch pool, or make them specify it? Are we comfortable with a situation where they might accidentally have their rgw metadata on a non-stretch pool that gets lost due to a DC failure?

While I can see a future where we want to do it on a per-pool basis, doing it on the whole cluster fits the initial use cases and is a lot simpler/safer.